ci/azure-pipelines-windows.yml

```diff
@@ -0,0 +1,38 @@
+# Template for the basic Windows steps in Azure Pipelines
+
+steps:
+
+- bash: echo "##vso[task.prependpath]$CONDA\Scripts"
+  displayName: Add conda to PATH
+
+- bash: |
+    set -x -e
+    conda config --set always_yes yes --set changeps1 no --set show_channel_urls true
+    conda install libnetcdf libcurl libgdal pcre
+    conda list
+  displayName: Install dependencies via conda
+
+- bash: echo "##vso[task.setvariable variable=INSTALLDIR]D:/a/1/s/gmt-install-dir"
```

Comment:
Here, I use `D:/a/1/s` rather than `$BUILD_SOURCEDIRECTORY` (D:\a\1\s).

If `$BUILD_SOURCEDIRECTORY` is used, then 
```
GMT->session.SHAREDIR = D:s/gmt-install-dir/share
```
and gmt will fail to find some files in share directory.